views/partials/tool/object-options.html

```diff
@@ -7,12 +7,12 @@
 
     <button class="btn btn-divider"></button>
 
-    <button class="btn btn-transparent btn-icon-labeled" v-on:click="toggle_lock_of_selected_artifacts()" v-if="active_space_role=='admin'" title="<%=__("lock")%>">
+    <button class="btn btn-transparent btn-icon-labeled" v-on:click="toggle_lock_of_selected_artifacts()" v-if="active_space_role=='admin' || toolbar_lock_in">
```

Comment:
please manually merge latest `mnt` branch changes, there are now `lock_selected_artifacts` and `unlock_selected_artifacts` functions to make the buttons work predictably.